src/pages/authentication-pages/signup/Signup.jsx

```diff
@@ -0,0 +1,182 @@
+import { useState } from "react";
+import { NavLink, useNavigate } from "react-router-dom";
+import { Navbar } from "../../../components/index";
+import axios from "axios";
+
+function Signup() {
+  const [userSignupData, setUserSignupData] = useState({
+    firstName: "",
+    lastName: "",
+    email: "",
+    password: "",
+    confirmPassword: "",
+  });
+
+  const navigate = useNavigate();
+
+  function handleSignup() {
+    const userData = {
+      email: userSignupData.email,
+      password: userSignupData.password,
+    };
+
+    (async () => {
+      try {
+        const response = await axios.post("/api/auth/signup", userData);
+        localStorage.setItem(
+          "token",
+          JSON.stringify(response.data.encodedToken)
```

Comment:
no need of stringify in this case, as token is already a string

Reply:
done